src/components/sidebar/routePathView/RoutePathView.tsx

```diff
@@ -194,6 +198,36 @@ class RoutePathView extends ViewFormBase<
                 e
             );
         }
+        try {
+          const routePathLinks: IRoutePathLink[] = this.props.routePathStore!.routePath!.routePathLinks;
+          const promises: any[] = [];
+          const kilpiViaNames: IKilpiVia[] = [];
+
+          routePathLinks.forEach((routePathLink: IRoutePathLink) => {
+            const createPromise = async () => {
+              try {
+                const kilpiViaName: IKilpiVia | null = await KilpiViaService.fetchKilpiViaName(routePathLink.id);
+                if (kilpiViaName) kilpiViaNames.push(kilpiViaName);
+              } catch (err) {
+                this.props.errorStore!.addError(
+                    'KilpiVia haku ei onnistunut.',
+                    err
+                )
+              }
```

Comment:
Fix indent, should be 4. Here it seems to be randomly 2-4. Strongly recommend installing prettier and getting it to work when saving (we already have it configured for the project if you look at package.json)